platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -87,6 +87,8 @@
                 dialog.getModalBody().css('padding', '0');
 
                 dialog.open();
+
+                dialog.getModal().prev().css('z-index', dialog.getModal().css('z-index'));
```

Comment:
@mkruszynski What kind of modal is used in Tasks? Doesn't it use our own ModalService, that was responsible for displaying only the latest modal?